vlib/v/fmt/tests/comments_expected.vv

```diff
@@ -104,47 +91,37 @@ fn between_if_branches() {
 	}
 }
 
-// comments
-for {
+for // comments
+{
```

Comment:
I am not sure about this ... the hanging `{` on its own line seems a little odd to me.